lib/Game.js

```diff
@@ -29,13 +30,16 @@ class Game {
     domUpdates.displayCategory(this.puzzle.category)
     this.puzzle.splitAnswer();
     domUpdates.displayRound(this.currentRound);
+    console.log(this.puzzle)
```

Comment:
We probably want to get this out of the console.log out before we merge to master.

Reply:
just pulled it out of the file. i'll commit it on the next round. Thanks!

Reply:
Cool sounds good!